### PR TITLE
Added check to avoid insertion of same starred messages & findMessage…

### PR DESCRIPTION
**Pull request by taneja1995**
…ById api

### CODE REVIEW

**Review by chayank-neu — APPROVED** [review]

File: controllers/MessageController.ts [hunk 1]
```diff
@@ -59,6 +60,11 @@ export default class MessageController implements MessageControllerI {
 
     }
 
+    findMessageById = (req:Request, res:Response) =>{
```
Comment (lalwanijayesh) [inline controllers/MessageController.ts:69]: Missing comments for JSdoc, can you please add that? LGTM otherwise.

**Review by lalwanijayesh — CHANGES_REQUESTED** [review]
Missing JSdoc for the new findMessageById API

File: controllers/MessageController.ts (hunk 1 above)
Comment (taneja1995) [reply controllers/MessageController.ts:69]: Added JSdoc for the new message api.

**Review by lalwanijayesh — APPROVED** [review]
Looks good

**Pull request merged**